api-docs/paths/releases/sessions.json

```diff
@@ -30,7 +32,7 @@
       {
         "name": "field",
         "in": "query",
-        "description": "The list of fields to query.\n\nThe available fields are `sum(session)`, `count_unique(user)`, and the following functions applied to the `session.duration` metric: `avg`, `p50`, `p75`, `p90`, `p95`, `p99` and `max`.\n\nFor example, `p99(session.duration)`.",
+        "description": "The list of fields to query.\n\nThe available fields are\n  - `sum(session)`\n  - `count_unique(user)`\n  - `avg`, `p50`, `p75`, `p90`, `p95`, `p99`, `max` applied to `session.duration`. For example, `p99(session.duration)`\n  - `crash_rate`, `crash_free_rate` applied to `user` or `session`. For example, `crash_free_rate(user)`",
```

Comment:
We also support min and sum over distribution metrics?

Reply:
Good point. They weren't supported in the sessions-based API so they also aren't part of the `FIELD_MAP`. I'm not opposed to extending it, but that would be out of scope for this PR.

https://github.com/getsentry/sentry/blob/de4fda7603dc83ad51f9b2be3b5d28db72257788/src/sentry/release_health/metrics_sessions_v2.py#L384-L398

Reply:
Preview: 
![image](https://user-images.githubusercontent.com/1565449/201616020-7387de5f-1891-464e-9c17-89282597a0d8.png)